sources/packages/backend/apps/api/src/services/education-program-offering/custom-validators/custom-max-value.ts

```diff
@@ -0,0 +1,17 @@
+import { MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE } from "../../../utilities";
+import {
+  ValidatorConstraint,
+  ValidatorConstraintInterface,
+  ValidationArguments,
+} from "class-validator";
+
+@ValidatorConstraint({ name: "CustomMaxValue", async: false })
```

Comment:
I noticed the other discussion around this decorator. If we will keep this please add some comments.
I also believe that in general, we create the function to have it decorated.